cost-analyzer/templates/_helpers.tpl

```diff
@@ -1421,6 +1429,22 @@ Groups is only used when using simple RBAC.
     {{- end }}
 {{- end }}
 
+{{- define "authMasterKeyEnabled" -}}
+  {{- if or (.Values.saml).enabled (.Values.oidc).enabled -}}
+    {{- if or (.Values.saml).apiMasterKey (.Values.oidc).apiMasterKey -}}
+      {{- printf "true" -}}
+    {{- else -}}
+      {{- if or (.Values.saml).apiMasterKeySecret (.Values.oidc).apiMasterKeySecret -}}
+        {{- printf "true" -}}
```

Comment:
We should probably document the usage of `apiMasterKey` or `apiMasterKeySecret`? At least in the `values.yaml` file?

Reply:
No undocumented values, guys.

Reply:
@kaelanspatel ☝️